10.Mender-server/docs.md

```diff
@@ -1316,6 +1316,319 @@ New changes in integration since 3.3.0:
+|---|---|
+| Lluis Campos | 1003 (21.4%) |
+| Peter Grzybowski | 843 (18.0%) |
+| Alf-Rune Siqveland | 821 (17.5%) |
+| Fabio Tranchitella | 677 (14.4%) |
+| Kristian Amlie | 674 (14.4%) |
+| Maciej Tomczuk | 236 (5.0%) |
+| Alex Miliukov | 236 (5.0%) |
+| Ole Petter Orhagen | 81 (1.7%) |
+| Mikael Torp-Holte | 67 (1.4%) |
+| Manuel Zedel | 50 (1.1%) |
+
+| Developers with the most lines removed | |
+|---|---|
+| Kristian Amlie | 188 (7.5%) |
+
+| Developers with the most signoffs (total 4) | |
+|---|---|
+| Ole Petter Orhagen | 2 (50.0%) |
+| Kristian Amlie | 1 (25.0%) |
+| Fabio Tranchitella | 1 (25.0%) |
+
+| Top changeset contributors by employer | |
+|---|---|
+| Northern.tech | 118 (100.0%) |
+
+| Top lines changed by employer | |
+|---|---|
+| Northern.tech | 4688 (100.0%) |
+
+| Employers with the most signoffs (total 4) | |
+|---|---|
+| Northern.tech | 4 (100.0%) |
+
+| Employers with the most hackers (total 10) | |
+|---|---|
+| Northern.tech | 10 (100.0%) |
+
+### Changelogs
+
+#### auditlogs (3.0.2)
+
+New changes in auditlogs since 3.0.1:
+
+* upgrade mendersoftware/go-lib-micro to v0.0.0-20220304124258-d8414885f1c3
+  ([MEN-5506](https://tracker.mender.io/browse/MEN-5506))
+
+#### deployments (4.2.1)
+
+New changes in deployments since 4.2.0:
+
+##### Bug Fixes
+
+* Fix bad indexing from database versions 1.2.8 and 1.2.9
+  ([MEN-5695](https://tracker.mender.io/browse/MEN-5695))
+* Escape configuration download URL path and support '.' in parameter
+  ([MEN-5712](https://tracker.mender.io/browse/MEN-5712))
+* Multipart upload not working when loading s3 settings from env
+  ([MEN-5939](https://tracker.mender.io/browse/MEN-5939))
+
+#### deployments-enterprise (4.2.1)
+
+New changes in deployments-enterprise since 4.2.0:
+
+##### Bug Fixes
+
+* Fix bad indexing from database versions 1.2.8 and 1.2.9
+  ([MEN-5695](https://tracker.mender.io/browse/MEN-5695))
+* Escape configuration download URL path and support '.' in parameter
+  ([MEN-5712](https://tracker.mender.io/browse/MEN-5712))
+* Multipart upload not working when loading s3 settings from env
+  ([MEN-5939](https://tracker.mender.io/browse/MEN-5939))
+
+#### deviceauth (3.2.2)
+
+New changes in deviceauth since 3.2.1:
+
+##### Bug Fixes
+
+* replace colon with hash for path params modifiers in URLs
+  ([MEN-5713](https://tracker.mender.io/browse/MEN-5713))
+* Missing hash in inventory client
+
+##### Other
+
+* upgrade mendersoftware/go-lib-micro to v0.0.0-20220302103938-7561c751e1fd
+  ([MEN-5471](https://tracker.mender.io/browse/MEN-5471))
+
+#### deviceauth-enterprise (3.2.2)
+
+New changes in deviceauth-enterprise since 3.2.1:
+
+##### Bug Fixes
+
+* replace colon with hash for path params modifiers in URLs
+  ([MEN-5713](https://tracker.mender.io/browse/MEN-5713))
+* replace colon with hash for path params modifiers in URLs
+  ([MEN-5713](https://tracker.mender.io/browse/MEN-5713))
+* Missing hash in inventory client
+* Missing hash in inventory client
```

Comment:
This duplicity came from a bad merge. I remember I took mental note to fix with `cancel-changelog` but the item never got into my TODO list. Please remove them manually from here.